support-workers/cloud-formation/src/templates/state-machine.yaml

```diff
@@ -68,6 +68,93 @@ States:
           Resource: "${SendAcquisitionEventLambda.Arn}"
           End: True
           {{> retry}}
+
+    # We use this step to pass through state that can then be read to three tier steps
+    - StartAt: ThreeTierPassThrough
+      States:
+        ThreeTierPassThrough:
+          Type: Pass
+          End: True
+    Next: ThreeTierUnpackJsonList
+
+  # Unpack the list of states from the parallel tasks into a single item
+  ThreeTierUnpackJsonList:
+    Type: Pass
+    InputPath: $.[0]
+    Next: ThreeTierShouldCreateSupporterPlusSubscription
+
+  ThreeTierShouldCreateSupporterPlusSubscription:
+    Type: Choice
+    Choices:
+      - Next: ThreeTierUnpackSendThankYouEmailState
+        And:
+          - Variable: $.state.acquisitionData.threeTierCreateSupporterPlusSubscription
+            IsPresent: true
+          - Variable: $.state.acquisitionData.threeTierCreateSupporterPlusSubscription
+            BooleanEquals: true
+          - Variable: $.state.sendThankYouEmailState.product.productType
+            IsPresent: true
+          - Variable: $.state.sendThankYouEmailState.product.productType
+            StringEquals: GuardianWeekly
+    Default: Done
+
+  # This transform
+  # - moves state from `sendThankYouEmailState` back into the root state
+  # - adds `paymentFields.billingAccountId` that's required by `PreparePaymentMethodForReuse`
+  ThreeTierUnpackSendThankYouEmailState:
+    Type: Pass
+    Parameters:
+      state:
+        paymentFields:
+          billingAccountId.$: $.state.sendThankYouEmailState.accountNumber
+        user.$: $.state.sendThankYouEmailState.user
+        product.$: $.state.sendThankYouEmailState.product
+        giftRecipient.$: $.state.sendThankYouEmailState.giftRecipient
+        paymentMethod.$: $.state.sendThankYouEmailState.paymentMethod
+        paymentSchedule.$: $.state.sendThankYouEmailState.paymentSchedule
+        promoCode.$: $.state.sendThankYouEmailState.promoCode
+        accountNumber.$: $.state.sendThankYouEmailState.accountNumber
+        subscriptionNumber.$: $.state.sendThankYouEmailState.subscriptionNumber
+        firstDeliveryDate.$: $.state.sendThankYouEmailState.firstDeliveryDate
+        productType.$: $.state.sendThankYouEmailState.productType
+        requestId.$: $.state.requestId
+        analyticsInfo.$: $.state.analyticsInfo
+        acquisitionData.$: $.state.acquisitionData
+      error.$: $.error
+      requestInfo.$: $.requestInfo
+    Next: ThreeTierChooseSubscriberPlusProduct
+
```

Comment:
Add a note here how the decision is being made.

Reply:
[Done](https://github.com/guardian/support-frontend/pull/5629/commits/c69aed8084e09f8f80b6a1e3569534556f7e473f)